packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -1,31 +1,55 @@
 {# API #}
 {%- set props = props | default([]) -%}
+{%- set _editText = props.editText | default('Edit') -%}
 {%- set _fileName = props.fileName | default(null) -%}
+{%- set _hasImagePreview = props.hasImagePreview | default(false) -%}
```

Comment:
This prop name is a little bit confusing to me — the attachment does not yet have the preview. The preview is yet to be generated by JS plugin.

Maybe `generateImagePreview` would be clearer? The pair of (mutually exclusive) `generateImagePreview` and `imagePreview` works better for me.

cc @literat